db/migrate/20231115112652_add_confirmable_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddConfirmableToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
+
+    add_index :users, :confirmation_token, unique: true
+  end
+end
```

Comment:
- [OPTIONAL] This migration file isn't needed because it is available in the 
`db/migrate/20231115003415_add_devise_to_users.rb` migration file. All you have to do is uncomment lines 25-28 of this migration file. 👍 😎